### DISCUSSION TITLE
Unable to scroll the feed when swiping on multipicture posts

### DISCUSSION THREAD

**Discussion by dimipage** [pixelfed/pixelfed-rn#16] [Bug Reports]
Swiping the feed doesnt work on multipic posts. Swipe action works left and right for picture scrolling but up and down doesnt scroll the feed.

**Comment by jampattern** [comment]
Same here, however, after you have swiped to the last image in a multi post, vertical swiping works again on that post. Pixel 4, Android 13.

**Comment by dansup** [comment]
Fixed in the latest commits!
